resources/commands.js

```diff
@@ -694,79 +694,75 @@ status.command({
             }
         );
 
-        return status.components.view({}, [text, image]);
+        return {markup: status.components.view({}, [text, image])};
     },
-    shortPreview: function(params) {
-        return status.components.text(
-            {},
-            I18n.t('location_title') + ": " + params.address
-        );
+    shortPreview: function (params) {
+        return {
+            markup: status.components.text(
+                {},
+                I18n.t('location_title') + ": " + params.address
+            )
+        };
     }
 }).param({
     name: "address",
     type: status.types.TEXT,
     placeholder: I18n.t('location_address')
 });
 
-
-function browseSuggestions(params) {
-    if (params.url && params.url !== "undefined" && params.url != "") {
-        var url = params.url;
-        if (!/^[a-zA-Z-_]+:/.test(url)) {
-            url = 'http://' + url;
-        }
-
-        return {webViewUrl: url};
-    }
-}
-
 status.command({
     name: "browse",
+    fullscreen: true,
     title: I18n.t('browse_title'),
     description: I18n.t('browse_description'),
-    color: "#ffa500",
-    fullscreen: true,
-    suggestionsTrigger: 'on-send',
     params: [{
         name: "url",
-        suggestions: browseSuggestions,
+        placeholder: "url",
         type: status.types.TEXT
-    }]
+    }],
+    onSend: function (params, context) {
+        var url = params.args.url;
```

Comment:
I looked at `:set-chat-input-metadata` and `::request-command-data` and both of them don't make clear what this `metadata` contains and how it can be useful in commands handlers inside the jail.